app/jobs/event_job.rb

```diff
@@ -9,10 +9,10 @@ def queue_name
     :event
   end
 
-  attr_accessor :generic_file_id, :depositor_id
+  attr_accessor :id, :depositor_id
 
-  def initialize(generic_file_id, depositor_id)
-    self.generic_file_id = generic_file_id
+  def initialize(id, depositor_id)
```

Comment:
I plan on sorting out how to get the event machinery to be agnostic as to GenericFile or GenericWork, and only delegating to specific behavior at the last possible module or class.